webapi/src/main/scala/org/knora/webapi/responders/v1/ResourcesResponderV1.scala

```diff
@@ -1176,66 +1176,6 @@ class ResourcesResponderV1 extends Responder {
 
 
     /**
-      * Implements a pre-update check to ensure that an [[UpdateValueV1]] has the correct type for the `knora-base:objectClassConstraint` of
```

Comment:
I would be grateful if you could have a look at this merge https://github.com/dhlab-basel/Knora/pull/478/commits/a9b6780d2fcad45123342e65b2771565d3c4ee0f